recipes/perfmetrics/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "perfmetrics" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/perfmetrics-{{ version }}.tar.gz
+  sha256: b2c2a348989bcd229dd3621645507bb3d9877f442f0cf937d33079db585a7529
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - perfmetrics
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zodb/perfmetrics
+  summary: Send performance metrics about Python code to Statsd
+  license: BSD-3-Clause
```

Comment:
That's not a BSD 3 clause license--it has a 4th clause. Not sure if that's standard or a one-off.

Reply:
Changed to `BSD-3-Clause-Modification`

https://spdx.org/licenses/BSD-3-Clause-Modification.html

Reply:
:+1: